bolt/test/X86/dwarf5-df-inlined-subroutine-gc-sections-range.test

```diff
@@ -6,7 +6,7 @@
 ; RUN: -split-dwarf-file=main.dwo -o main.o
 ; RUN: llvm-mc -dwarf-version=5 -filetype=obj -triple x86_64-unknown-linux %p/Inputs/dwarf5-df-inlined-subroutine-gc-sections-range-helper.s \
 ; RUN: -split-dwarf-file=helper.dwo -o helper.o
-; RUN: %clang --target=x86_64-pc-linux -fuse-ld=lld -Wl,-gc-sections -Wl,-q -gdwarf-5 -gsplit-dwarf=split main.o helper.o -o main.exe
+; RUN: %clang -fuse-ld=lld -Wl,-gc-sections -Wl,-q -gdwarf-5 -gsplit-dwarf=split main.o helper.o -o main.exe
```

Comment:
Why are you dropping the `--target=` here?

Reply:
From the commit message:
> The X86 directory lit config sets it already, there's no reason for this test to do it by itself.

Notably this is also the only test in the whole test suite that uses the `x86_64-pc-linux` target and doesn't seem to actually test anything related to that. On my downstream reasons this also causes the test to fail for reasons irrelevant to upstream LLVM, but the change is IMO justified in of itself still and was simply revealed because of that.